### ISSUE TITLE
Broken link to PP-OCR Model Library in finetune documentation

### ISSUE THREAD

**Issue by zero-stroke** [PaddlePaddle/Paddle#75922]
### 文档链接&描述 Document Links & Description

In the docs at https://www.paddleocr.ai/main/en/version2.x/ppocr/model_train/finetune.html?h=fine#21-dataset it says to go to the PP-OCR Series Model Library to view more models, but the link is broken. 

> It is recommended to choose the PP-OCRv3 model (configuration file: [PP-OCRv3_mobile_det.yml](https://github.com/PaddlePaddle/PaddleOCR/tree/main/configs/det/PP-OCRv3/PP-OCRv3_mobile_det.yml)，pre-trained model: [ch_PP-OCRv3_det_distill_train.tar](https://paddleocr.bj.bcebos.com/PP-OCRv3/chinese/ch_PP-OCRv3_det_distill_train.tar), its accuracy and generalization performance is the best pre-training model currently available.
> 
> For more PP-OCR series models, please refer to [PP-OCR Series Model Library](https://www.paddleocr.ai/main/en/version2.x/ppocr/models_list.en.md).

Where could I go to find more pre-train models to download for fine-tuning, specifically where could I download en_PP-OCRv5_rec?

**Comment by Bobholamovic** [comment]
You can find it in this document: https://www.paddleocr.ai/latest/en/version3.x/pipeline_usage/OCR.html#1-ocr-pipeline-introduction .

For en_PP-OCRv5_mobile_rec, the link is https://paddle-model-ecology.bj.bcebos.com/paddlex/official_inference_model/paddle3.0.0//en_PP-OCRv5_mobile_rec_infer.tar .
